src/routes/living-form/containers/NeuburgForm.js

```diff
@@ -0,0 +1,216 @@
+            <LawParagraph>
+              Hierzu stellt die Tür an Tür - Digital Factory gGmbH meine
+              personenbezogenen Daten auf die öffentliche App Integreat, die vom
+              Landkreis Neuburg-Schrobenhausen genutzt wird, um Wohnungen und
+              Mietobjekte zielgerichtet zu vermitteln.
+            </LawParagraph>
+            <LawParagraph>
+              Die Einwilligung kann verweigert bzw. jederzeit ohne Angabe von Gründen über den in der Bestätigungsmail
+              versendeten Link zur dauerhaften Löschung widerrufen werden. In diesem
+              Fall erfolgt keine Bearbeitung meines Mietangebots; alle gespeicherten
+              personenbezogenen Daten werden gelöscht.
+            </LawParagraph>
+          </WideCol>
+        </Row>
+        <Row style={{margin: '20px 0', justifyContent: 'center'}}>
+          <Col center='true'>
+            <FormControl>
+              <SubmitButton validateAll={this.validateAll} type='submit' disabled={this.props.sending}>
+                {this.props.sending ? 'Wird gesendet...' : 'Mietangebot Senden'}
+              </SubmitButton>
+              {this.props.serverError && <FormHelperText error>{this.props.serverError}</FormHelperText>}
+            </FormControl>
+          </Col>
+        </Row>
+        <Row>
+          <StdCol>
+            Dieses Projekt wird aus Mitteln des Bayerischen Staatsministeriums für Arbeit und Soziales, Familie und
+            Integration gefördert.
+          </StdCol>
+          <StdCol>
+            <img src={staatLogo} style={{width: '100%'}} />
+          </StdCol>
+        </Row>
+      </Grid>
+    </Form>
+  }
+
+  handleSubmit = event => {
+    event.preventDefault()
+    const values = this.state.form.getValues()
+    // Transform flat object with properties including a dot to nested object
+    const requestBody = {}
+    forEach(values, (value, key) => set(requestBody, key, value))
+    // Make ofAdditionalCosts, ofRooms, ofAdditionalServices arrays (if not already)
+    if (!Array.isArray(requestBody.formData.accommodation.ofRooms)) {
+      requestBody.formData.accommodation.ofRooms = [requestBody.formData.accommodation.ofRooms].filter(key => key)
+    }
+    if (!Array.isArray(requestBody.formData.costs.ofAdditionalServices)) {
+      requestBody.formData.costs.ofAdditionalServices = Array.of(requestBody.formData.costs.ofAdditionalServices).filter(key => key)
+    }
+    if (!Array.isArray(requestBody.formData.costs.ofRunningServices)) {
+      requestBody.formData.costs.ofRunningServices = Array.of(requestBody.formData.costs.ofRunningServices).filter(key => key)
+    }
+    // Convert boolean values to actual bools
+    requestBody.agreedToDataProtection = requestBody.agreedToDataProtection === 'true'
+    requestBody.formData.costs.hotWaterInHeatingCosts = requestBody.formData.costs.hotWaterInHeatingCosts === 'true'
+
+    this.props.sendRequest(requestBody)
+  }
+}
+
+export default NeuburgForm
```

Comment:
This is quite big, though I think this is okay since it does not have to be reusable.